pkg/fleetautoscalers/controller.go

```diff
@@ -152,6 +148,18 @@ func NewController(
 	return c
 }
 
+// NewExtensions binds the handlers to the webhook outside the initialization of the controller
+// initializes a new logger for extensions.
+func NewExtensions(wh *webhooks.WebHook) {
+	extensionLogger = runtime.NewLoggerWithSource("FleetAutscalerExtensions")
```

Comment:
Creating a global variable to share state between this function and the validation function seems less than ideal. 

What about creating a struct that just holds the logger and hanging the mutationHandler function off of it (similarly to how it was previously hanging off of the controller struct)?

(this same comment applies to the other controller files as well)

Reply:
There is now a `Extensions` struct that contains the logger. The `Extensions` struct is now the receiver for the handler functions as well.